src/include/cc.h

```diff
@@ -83,12 +86,45 @@ struct Pcb {
   }
   void rto_advance() { rto_timer++; }
 
+  void sack_bitmap_shift_right_one() {
+    auto sack_bitmap_bucket_max_idx = kSackBitmapSize/sizeof(sack_bitmap[0]) - 1;
```

Comment:
Type-inferencing is doing a lot of heavy lifting below (e.g., the fact that some are references was unclear). Best to make it explicit instead to make it more readable. See https://google.github.io/styleguide/cppguide.html#Type_deduction

* `constexpr size_t sack_bitmap_bucket_max_idx`
* `uint64_t &sack_bitmap_right_bucket`
* `const uint64_t sack_bitmap_left_bucket`
* `for (uint8_t i` -> `for (size_t i` 
* `uint64_t &sack_bitmap_left_most_bucket`

We should remove all uses of `auto` in this PR.